src/Workspaces/Core/Portable/Utilities/TaskQueue.cs

```diff
@@ -74,8 +74,7 @@ public Task<T> ScheduleTask<T>(string taskName, Func<Task<T>> operation, Cancell
         /// </summary>
         /// <returns>The <see cref="Task"/> that executes the operation.</returns>
```

Comment:
I think we can now remove these comments and rename the methods to `ScheduleTaskImpl`

Reply:
Sure :)